src/components/comments/dropdown-menu.tsx

```diff
@@ -88,14 +88,14 @@ export function DropdownMenu({
   function onDelete() {
     onAnyClick()
     if (!isParent) {
-      void setCommentState({ id, trashed: true })
+      void setCommentState({ id: [id], trashed: true })
```

Comment:
Is this necessary? Shouldn't GraphQL accept a single id, too?

Reply:
It accepts single id's but the type does not. Maybe a problem with `regenerate-types`?

Reply:
Interesting. That's kinda annoying. Feel free to create an issue in api.serlo.org for that. Not sure whether there is an easy fix though since we just extract that from the GraphQL schema (so possibly a feature request for the library that does the extraction).

Reply:
Jup, I kind of wish I knew that before I [advocated this](https://graphql-rules.com/rules/mutation-batch-changes).

Reply:
follow up in https://github.com/serlo/api.serlo.org/issues/230